WordPress/src/main/java/org/wordpress/android/ui/posts/services/MediaUploadReadyProcessor.java

```diff
@@ -22,10 +23,9 @@ public PostModel replaceMediaFileWithUrlInPost(@Nullable PostModel post, String
                         localMediaId, mediaFile);
                 post.setContent(modifiedContents);
             } else if (showNewEditor) {
-                // TODO implement visual editor implementation to update image/video in post
-            } else {
-                // TODO implement legacy editor implementation to update image/video in post
+                post.setContent(EditorFragment.replaceMediaFileWithUrl(post.getContent(), mediaFile));
```

Comment:
I like this is good, can we also eliminate the intermediate `modifiedContents` variable up here in [line 22](https://github.com/wordpress-mobile/WordPress-Android/pull/5833/files#diff-72b7d5dc037579f785245973ecad6923R22)? Not part of your PR but just saying we could replicate the same way in there